main/src/ca/uwaterloo/flix/language/ast/TypedAst.scala

```diff
@@ -43,7 +43,8 @@ object TypedAst {
                   sources: Map[Source, SourceLocation],
                   traitEnv: Map[Symbol.TraitSym, TraitContext],
                   eqEnv: ListMap[Symbol.AssocTypeSym, shared.AssocTypeDef],
-                  names: MultiMap[List[String], String],
+//                  names: MultiMap[List[String], String],
+                  availableClasses: AvailableClasses,
```

Comment:
This line should also be added to the previous ASTs:

- ParsedAst
- ...
- NamedAst
- ResolvedAst
- KindedAst

Reply:
You can see how "names" is used-- which should be deleted.